### ISSUE TITLE
PSM2 MTL fails to compile on master

### ISSUE THREAD

**Issue by hppritcha** [open-mpi/ompi#8666]
I'm getting this error trying to build Open MPI on a system with PSM2 installed using GCC 9.3.0
```
mtl_psm2.c: In function ‘ompi_mtl_psm2_add_procs’:
mtl_psm2.c:307:23: error: implicit declaration of function ‘opal_max’ [-Werror=implicit-function-declaration]
  307 |     timeout_in_secs = opal_max(ompi_mtl_psm2.connect_timeout, 0.5 * nprocs);
```

**Comment by hppritcha** [comment]
this seems to have been magically fixed by a rebase on top of master at sha 8b273777685cd41ef26021e91e1ff077d9dd0699

**Comment by mwheinz** [comment]
JFC, take a long weekend to relax and recover from a year of lock down and months of physical therapy and all heck breaks loose. ;-)
